arch/arm/soc/nordic_nrf/nrf52/Kconfig.series

```diff
@@ -8,7 +8,7 @@
 config  SOC_SERIES_NRF52X
 	bool "Nordic Semiconductor nRF52 series MCU"
 	select CPU_CORTEX_M4
-	select CPU_HAS_FPU
+	select CPU_HAS_FPU if !SOC_NRF52810
```

Comment:
At this point, seems like move the 'select CPU_HAS_FPU' over to the SOC_NRF52... would be better.

Reply:
Done